cycode/cli/main.py

```diff
@@ -111,6 +125,7 @@ def code_scan(
     sca_scan: List[str],
     monitor: bool,
     report: bool,
+    no_restore: bool,
```

Comment:
sca_no_restore

Reply:
flag name must be equal to parameters name